src/main/cluster.ts

```diff
@@ -416,6 +417,10 @@ export class Cluster implements ClusterModel {
   }
 
   protected async getAllowedNamespaces() {
+    if (this.accessibleNamespaces?.length) {
```

Comment:
If we'll set empty initial value to `accessibleNamespaces` (see my comment above), we can skip `?` operator.